iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -251,10 +251,13 @@ static bool isDispatchableOp(Operation *op) {
   if (op->getParentOfType<IREE::Flow::DispatchWorkgroupsOp>()) {
     return false;
   }
+  // subtensor/subtensor_insert/scf.for ops are dispatchable.
+  if (isa<scf::ForOp, SubTensorOp, SubTensorInsertOp>(op)) {
```

Comment:
This is not the same logic as below. If the op is a `subtensorOp` or `subTensorInsertOp` it will return what `!isAlwaysCloneIntoDispatchOp`. I think you can just add `scf.for` to the list with `SubTensor` and `SubTensorInsertOp` where it is currently.

THough in this case, it makes no difference